src/pages/home/ReportScreen.js

```diff
@@ -184,7 +189,7 @@ class ReportScreen extends React.Component {
                     style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
                 >
                     {this.shouldShowLoader() && <FullScreenLoadingIndicator />}
-                    {!this.shouldShowLoader() && (
+                    {!this.shouldShowLoader() && this.state.composerMounted && (
```

Comment:
So we will wait for the text input to mount... but why? Can you please explain this both in the PR's description and probably somewhere as a code comment? It's not clear why we need to do this.

Reply:
I will add the comments when the solution is finalized.